tests/serverpod_test_server/lib/src/generated/entities_with_list_relations/city.dart

```diff
@@ -144,21 +147,26 @@ abstract class City extends _i1.TableRow {
       orderDescending: orderDescending,
       useCache: useCache,
       transaction: transaction,
+      include: include,
     );
   }
 
   @Deprecated('Will be removed in 2.0.0. Use: db.findById instead.')
   static Future<City?> findById(
     _i1.Session session,
-    int id,
-  ) async {
-    return session.db.findById<City>(id);
+    int id, {
+    CityInclude? include,
```

Comment:
Question: Is it necessary to add this for the old interface? I'm a bit confused around what paths are shared with the new code.

Reply:
I think it makes sense to add it as we already support include in them. They share most of the logic tbh, the major difference is the insert and update and delete methods.